crates/red_knot_python_semantic/src/types/narrow.rs

```diff
@@ -87,9 +87,9 @@ fn all_negative_narrowing_constraints_for_expression<'db>(
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum KnownConstraintFunction {
     /// `builtins.isinstance`
-    IsInstance,
+    Isinstance,
     /// `builtins.issubclass`
-    IsSubclass,
+    Issubclass,
```

Comment:
Hmm, I find this now much harder to read (I can see that it now matches the builtins name but I sort of prefer the old spelling)

Reply:
Bah! I'll switch it back and add a `#[strum(serialize = "issubclass")]` attribute to the variant 😄